lib/commands/clipboard.js

```diff
@@ -0,0 +1,99 @@
+import { exec } from 'teen_process';
+import { errors } from 'appium/driver';
+import { requireArgs } from '../utils';
+import _ from 'lodash';
+
+const commands = {};
+
+const CONTENT_TYPE = Object.freeze({
+  TEXT: 'text',
```

Comment:
`plaintext`, or `text`? (just for naming convention across drivers

Reply:
changed to plaintext